girder/models/user.py

```diff
@@ -46,6 +47,11 @@ def initialize(self):
         self.exposeFields(level=AccessType.ADMIN, fields=(
             'size', 'email', 'groups', 'groupInvites'))
 
+        events.bind('model.user.save.created', 'grantSelfAccess',
```

Comment:
We had discussed simply removing this and modifying `AccessControlledModel.hasAccess` to always return `True` if a user is checking access on themselves. Did you decide against making that change?